app/_src/guides/otel-metrics.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Collect metrics with OpenTelemetry  
+---
+
+{{site.mesh_product_name}} provides integration with [OpenTelemetry](https://opentelemetry.io/). You can collect and push 
+dataplane proxy and application metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/). Which opens up
+lots of possibilities of processing and exporting metrics to multiple ecosystems like [Datadog](https://www.datadoghq.com/),
+[Grafana cloud](https://grafana.com/products/cloud/), [Honeycomb](https://www.honeycomb.io/) and more.
```

Comment:
should we mention specific implementations or just link to the exporters listing https://opentelemetry.io/ecosystem/registry/?component=exporter ?

Reply:
we have this link later on when explaining configuration, I think that it should be enough